### DISCUSSION TITLE
OnRenderImage Attribute

### DISCUSSION THREAD

**Discussion by Deytooh** [naverz/zepeto-studio-global#20] [Q&A]
Hi ~~ :)

I would like to use an attribute called "ImageEffectOpaque" on OnRenderImage, I saw it was in the zepeto package, so I try to do like the @SerializedField() but doesnt really work like that I guess, so I tried to do like in Unity [ImageEffectOpaque] but not this way neither, does someone know how to use it with typescript ?

**Comment by jhslee** [comment]
Hello @Deytooh 👋

Unfortunately zepeto script does not support too many attribute tags quite yet. The currently supported attributes are as follows:

- NonSerialized
- HideInInspector
- SerializeField
- Header
- Space
- Tooltip

I apologize it took a while for the response, thank you for waiting, hopefully this hasn't caused too much of a trouble on your end, and ask for your understanding on this issue. 🥲

**Reply by Deytooh** [reply]
Hi ! Ohh I see, sad :( was trying to make a fog shader but will be complicated without this !
Thank you for the answer :) Have a nice day !
